specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2025-07-01-preview/PreviewAlertRule.json

```diff
@@ -0,0 +1,510 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2025-07-01-preview",
+    "title": "Preview Alert Rule API in Alerts Management Service Resource Provider",
+    "description": "The Preview Alert Rule API provides the ability to retrieve the results of a simulated historical execution of an alert rule"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceUri}/providers/Microsoft.AlertsManagement/previewAlertRule": {
```

Comment:
>"/{resourceUri}/providers/Microsoft.AlertsManagement/previewAlertRule" [](http://example.com/codeflow?start=4&length=70)

You are moving this API to a very wide-open scope. Is your API really able to extend any and every ARM resource?